.github/CODEOWNERS

```diff
@@ -10,6 +10,7 @@ yarn.lock                                           @backstage/community-plugins
 
 /workspaces/adr                                     @backstage/community-plugins-maintainers  @kuangp
 /workspaces/azure-devops                            @backstage/community-plugins-maintainers  @awanlin
+/workspaces/azure-storage-explorer                  @backstage/community-plugins-maintainers  @deepan10
```

Comment:
to fix this issue open an organization member request here: https://github.com/backstage/community/issues/new?assignees=&labels=org-member-request&projects=&template=org_member.yaml&title=Org+Member%3A+%3Cyour-github-login%3E

Reply:
Thanks! I am already part of the `backstage` organization

Reply:
Also added `packages/app` and `packages/backend`